core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -1686,12 +1689,23 @@ private void processBind(long lo, long msgLimit, @Transient SqlCompiler compiler
         validateParameterCounts(parameterFormatCount, parameterValueCount, parsePhaseBindVariableCount);
 
         lo += Short.BYTES;
-        if (parameterValueCount > 0) {
-            if (this.parsePhaseBindVariableCount == parameterValueCount) {
-                lo = bindValuesUsingSetters(lo, msgLimit, parameterValueCount);
-            } else {
-                lo = bindValuesAsStrings(lo, msgLimit, parameterValueCount);
+
+        try {
+            if (parameterValueCount > 0) {
+                if (this.parsePhaseBindVariableCount == parameterValueCount) {
+                    lo = bindValuesUsingSetters(lo, msgLimit, parameterValueCount);
+                } else {
+                    lo = bindValuesAsStrings(lo, msgLimit, parameterValueCount);
+                }
+            }
+        } catch (SqlException e) {
+            //if statement is named or portal then we've to add it to typesAndSelect
```

Comment:
nit: this comment doesn't make sense to me considering what's done in the below code, so we should either improve or delete it. WDYT?

Reply:
I think it's better to remove it because code removes typesAndSelect regardless of it being named or not . 
I checked that it still works even though in failed bind phase it removes typeAndSelect and triggers parsing on next execution .